### PR TITLE
Fixed "Intervals per query" panel in the "Mimir / Queries" dashboard

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
In #385 I've introduced "split_by_interval_and_results_cache" middleware, which then took over the old one:
https://github.com/grafana/mimir/pull/385/files#diff-79d26f5440b74049ba76e0fd11b44a321eaa0ea322a6d651d632341e81f6863aR188

However, this change has broken the "Intervals per Query" panel in the "Mimir / Queries" dashboard. This PR fixes it.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 ### Mixin
 
+* [BUGFIX] Dashboards: fixed "Intervals per Query" panel in the "Mimir / Queries" dashboard. #2308
```
Comment (osg-grafana) [inline CHANGELOG.md:12]:
```suggestion
* [BUGFIX] Dashboards: fixed "Intervals per query" panel in the "Mimir / Queries" dashboard. #2308
```
Please change Q to q in the UI.

**Review by osg-grafana — APPROVED** [review]
Approving with one nit about conforming to style.

File: CHANGELOG.md (hunk 1 above)
Comment (pracucci) [reply CHANGELOG.md]: Done!

**Review by jhesketh — APPROVED** [review]
lgtm

**Review by colega — APPROVED** [review]
LGTM, thanks

**Pull request merged**